### ISSUE TITLE
Questions regarding moderation

### ISSUE THREAD

**Issue by TheAlphaNerd** [nodejs/inclusivity#16]
It has been a crazy couple days and I have seen a non trivial number of comments deleted, but only mention of two individuals who have had any sort of follow up.

I think it is important for us to lock down a plan regarding how warning / banning is moderated, and that information to be publicly available.

It also feels potentially problematic that questions regarding moderation are currently being tabled within this WG. I'm not trying to WG creep, but perhaps inclusivity and moderation should be not be a single group. I would personally love to see this group focus on ways to expand the community, and while at times that may include moderation, I would hate to see the primary focus of this group become moderation.

There will be many needs for moderation
* irc
* gitter
* github
* conferences
* mailing lists
* one-on-one personal interactions

I would love to see a group focused on inclusivity be able to focus on positive ways to include more people in the project, and not need to get bogged down in the negativity in moderation. In the same light I would like to see any group focusing on moderation making informed decisions based on being inclusive.

TLDR; moderation should likely be it's own working group

**Comment by mikeal** [comment]
> It also feels potentially problematic that questions regarding moderation are currently being tabled within this WG.

The future process is being deferred to this group to take up on their first meeting but in the meantime we are moderating and banning and that is taking place in a private repo which is why you don't see it :)

**Comment by TheAlphaNerd** [comment]
It is also worth noting that with my many years of experience working with many communities I feel I could make strong and meaningful contributions to a moderation WG. Not being from an underrepresented group I would like to leave the podium of the inclusivity working group to those from under represented groups in order to have their voices heard

**Comment by TheAlphaNerd** [comment]
> The future process is being deferred to this group to take up on their first meeting but in the meantime we are moderating and banning and that is taking place in a private repo which is why you don't see it :)

My concern was less with transparency and more with this working group losing focus and becoming a moderation group

**Comment by mikeal** [comment]
> My concern was less with transparency and more with this working group losing focus and becoming a moderation group

Sure, but creating an inclusive moderation policy is one of the many things that should fall on this group and some subset of this group will hopefully want to help with the followup of enforcement of that policy. This is just jumping to the top of the list because we now have immediate moderation needs :)

**Comment by jasnell** [comment]
That's really why I created the nodejs/moderation private group and want to get the mail alias set up as quickly as possible. That gives those issues a home while allowing this WG to keep moving forward. That said, we need to make sure that our moderation policy is defined and documented.

**Comment by juliepagano** [comment]
I think it makes sense for this group to work on recommendations for moderation guidelines and potentially provide feedback/guidance to moderators, but not to actually run moderation itself.

**Comment by TheAlphaNerd** [comment]
@juliepagano thanks for nailing in one sentence what I couldn't in three paragraphs :D

**Comment by nebrius** [comment]
I like this idea @TheAlphaNerd. I think it makes sense for the inclusivity WG to keep handling it for now, but I think we should try and get the moderation WG set up soon.

**Comment by nebrius** [comment]
Wow, a lot has happened in the last couple months! We now have official [guidelines for moderation](https://github.com/nodejs/TSC/blob/master/Moderation-Policy.md) at the TSC level that I think covers most of this issue, and I think https://github.com/nodejs/inclusivity/issues/79 covers the rest, so I'm going to go ahead and close this issue.

**Comment by nebrius** [comment]
Oh, and to add: if there's anything that anyone here feels isn't covered by #79 and the TSC guidelines, please feel free to check out our new [issue guidelines](https://github.com/nodejs/inclusivity/blob/master/docs/POLICY_ISSUES.md) open a new issue.
